mooncake-store/src/client.cpp

```diff
@@ -259,7 +261,7 @@ std::optional<std::shared_ptr<Client>> Client::Create(
     const std::string& protocol, void** protocol_args,
     const std::string& master_server_entry) {
     auto client = std::shared_ptr<Client>(
-        new Client(local_hostname, metadata_connstring));
+        new Client(local_hostname, metadata_connstring, protocol));
```

Comment:
We should probably use an environment to open this, as I think it's still experimental.